pkg/virt-handler/migration-proxy/migration-proxy.go

```diff
@@ -343,7 +343,7 @@ func (m *migrationProxyManager) StopSourceListener(key string) {
 	}
 }
 
-// SRC POD ENV(migration unix socket) <-> HOST ENV (tcp client) <-----> HOST ENV (tcp server) <-> TARGET POD ENV (libvirtd unix socket)
+// SRC POD ENV(migration unix socket) <-> HOST ENV (tcp client) <-----> HOST ENV (tcp server) <-> TARGET POD ENV (virtqemud unix socket)
```

Comment:
Nit; `virtqemud`, maybe we can just use `migration` unix socket

Reply:
I think that this comment should describe the flow, hence `virtqemud` is more accurate than generic `migration`. WDYT?

Reply:
nit: maybe but at least we need to make consistency here, see the source("migration")